### DISCUSSION TITLE
The best way to check if a record exists (JS)

### DISCUSSION THREAD

**Discussion by xrenvtomate** [pocketbase/pocketbase#4045] [Q&A]
I want to create a user if there is no one with such email. If I just try to fetch one with this email I can get exception, is there an easier way of handling it? Or how to handle that exact exception

**Comment by ganigeorgiev** [comment]
> I want to create a user if there is no one with such email.

I don't think you need to execute any additional requests. The create action has builtin validator that will return an error if you try to create a user with an email that already exists.
To inspect the error response you can wrap your request with `try/catch` and `console.log(err.response)` (see also [jS SDK - Error handling](https://github.com/pocketbase/js-sdk?tab=readme-ov-file#error-handling)).
